source/sessions/driver-sessions.rst

```diff
@@ -751,19 +776,23 @@ The server response has the following format:
 
 .. code:: typescript
 
-    { ok : 1 }
+    interface EndSessionResponse {
+      ok: 1;
+    }
 
 In case of an error, the server response has the following format:
 
 .. code:: typescript
 
-    { ok : 0, errmsg : "...", code : NN }
+    interface EndSessionError {
+      ok: 0;
+      errmsg: string;
+      code: number;
+    }
 
 Drivers MUST ignore any errors returned by the ``endSessions`` command.
 
-Drivers that do not implement a server session pool MUST run the ``endSessions``
-command when the ``ClientSession.endSession`` method is called. Drivers that do
-implement a server session pool SHOULD run the ``endSessions`` command once when
+Drivers that implement a server session pool SHOULD run the ``endSessions`` command once when
```

Comment:
Remove "that implement a server session pool" since it is required now.

Reply:
Done, lmk if the english here reads good 😁